packages/react/src/components/autocomplete/autocomplete-option.tsx

```diff
@@ -3,6 +3,7 @@ import type { CSSUIObject, FC } from "../../core"
 import type { UseAutocompleteOptionProps } from "./use-autocomplete-option"
 import { ui } from "../../core"
 import { cx } from "../../utils"
+import { CheckIcon } from "../icon/icons"
```

Comment:
Shouldn't we first remove the icons that are deprecated since we will have to change the import path again?

Reply:
If you jump the code, you'll see that this is a Lucide icon, so there's no need to delete it.

Reply:
Shouldn't the writing style be consistent?

Reply:
How is it?

```diff
- import { CheckIcon } from "../icon/icons"
+ import { CheckIcon } from "../icon/"
```

Reply:
That is good.
However, the depercated icon in `icon/icons.tsx` takes priority, so you must remove it before making this change.

Reply:
I understand.
But if we remove that deprecated icon, we will have to deal with everything that uses it, so I think we should just merge it as it is now and remove the deprecated one after all the replacements are done and then unify the paths.

Reply:
@illionillion 
OK.
Don't forget to change the import path.